chart/values.yaml

```diff
@@ -58,6 +58,9 @@ config:
   awsRegion: ""
   awsSecretAccessKey: ""
   baaahThreadiness: "20"
+  bootstrapToken: "" # Randomly generated at runtime if not set
+  enableAuthentication: true
+  enableBootstrapUser: true
```

Comment:
what happen if you don't define `enableBootstrapUser`? will that be false and disable the user? 

I feel like it is more intuitive to use `disableBoottrapUser` and default to false so that user don't accidently disable that in helm chart

Reply:
ok, that's fine then.